config/jobs/image-pushing/k8s-staging-release-test.yaml

```diff
@@ -1,3 +1,77 @@
+periodics:
+- interval: 1h
+  name: ci-kubernetes-shadow-build
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: release
+    base_ref: master
+    path_alias: "k8s.io/release"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/image-builder:v20190927-21e0205
+      command:
+        - /run.sh
+      args:
+        - --project=k8s-staging-release-test
+        - --scratch-bucket=gs://k8s-staging-release-test-gcb
+        - --gcb-config=build.yaml
+        - --variant=build-ci
+        - --no-source
+        - gcb/
+      env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /creds/service-account.json
+      volumeMounts:
+        - name: creds
+          mountPath: /creds
+    volumes:
+      - name: creds
+        secret:
+          secretName: deployer-service-account
+  annotations:
+    fork-per-release: "true"
+    fork-per-release-generic-suffix: "true"
+    testgrid-dashboards: sig-release-shadow-master-blocking
```

Comment:
The use of the word "shadow" seems a bit opaque once you come at this PR from the perspective of seeing it on testgrid.  Is there a more plainly descriptive name it might have?

Reply:
Maybe sig-release-prototype-master-blocking?

Reply:
I like that. Will make the changes shortly.